lib/operations/distinct.js

```diff
@@ -73,14 +66,24 @@ class DistinctOperation extends OperationBaseV2 {
       return callback(err, null);
     }
 
-    // Execute the command
-    executeCommand(coll.s.db, cmd, options, (err, result) => {
```

Comment:
Do we not need any of the functionality from `executeCommand` anymore? I'm thinking of the following:
```
if (db.serverConfig && db.serverConfig.isDestroyed())
    return callback(new MongoError('topology was destroyed'));

// Debug information
  if (db.s.logger.isDebug())
    db.s.logger.debug(
      `executing command ${JSON.stringify(
        command
      )} against ${dbName}.$cmd with options [${JSON.stringify(
        debugOptions(debugFields, options)
      )}]`
    );
```